lib/travis/build/addons/firefox.rb

```diff
@@ -68,13 +69,32 @@ def export_source_url
               'firefox-beta-latest'
             when 'latest-esr'
               'firefox-esr-latest'
+            when 'latest-dev'
+              'firefox-aurora-latest'
+            when 'latest-nightly'
+              if RbConfig::CONFIG['host_os'] == /linux/
```

Comment:
This does not work. The bash script is compiled on a separate machine, which means that the platform detection must be done via the compiled bash script, not by `travis-build`.